### DISCUSSION TITLE
用Excel進行數據分析——分析股票行情

### DISCUSSION THREAD

**Discussion by williamyen0623** [1101-GS4719-C/Python-Community-GS4719-C#731] [科技新知]
這裡介紹了如何用Excel對股票進行數據分析，屬於比較進階的技術，大家可以參考看看，文章中也有提到Tushare這個免費、開源的python財經數據介面包。主要實現對股票等金融數據從數據採集、清洗加工 到 數據存儲的過程，能夠為金融分析人員提供快速、整潔、和多樣的便於分析的數據，為他們在數據獲取方面極大地減輕工作量，使他們更加專註於策略和模型的研究與實現上。考慮到Python pandas包在金融量化分析中體現出的優勢，Tushare返回的絕大部分的數據格式都是pandas DataFrame類型，非常便於用pandas/NumPy/Matplotlib進行數據分析和可視化。

我覺得要用AI來預測股價顯然不太可行，但是在量化交易、提高勝率、預測行情的部分還是蠻可靠的一個方法。

文章網址：
https://www.getit01.com/p2018081540197112/

**Comment by manipulateed** [comment]
太方便了吧，讓數據分析的難度先降低了一個門檻，也讓人有輕鬆的管道接觸這個領域ˋ。

**Comment by danwan1027** [comment]
這個太方便了吧！感謝推薦！

**Comment by Yunhans** [comment]
我完全不會用excel做圖。
好像用matplotlib也能做出差不多的表格？

**Comment by Pietra1226** [comment]
超厲害！原來excel有這麼多種功能，看來真的要去學了

**Comment by evanchou1204** [comment]
隨手可得的excel竟然有這個功能，太神啦

**Comment by CPCpyt** [comment]
我也常用EXCEL管理股票，
這樣的應用也帶給我些許啟發，能否運用在我的倉位中，
感謝分享！
